python/tvm/relax/frontend/stablehlo/stablehlo_translator.py

```diff
@@ -0,0 +1,431 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=invalid-name, inconsistent-return-statements, unidiomatic-typecheck
+# pylint: disable=import-outside-toplevel, unused-argument, no-member
+"""StableHLO frontend of Relax."""
```

Comment:
should we make this file pylint compliant?

Reply:
Good idea, have removed most of them now. Still have two: import-outside-toplevel, unused-argument. The first one is due to we import stablehlo and mlir inside the function, the second one is the input_info has not been used at this moment, will use it to feed input shapes (static/symbolic) for improvement.